modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -103,6 +103,8 @@ public class JavaClientCodegen extends AbstractJavaCodegen
 
     public static final String GENERATE_CLIENT_AS_BEAN = "generateClientAsBean";
 
+    public static final String REST_TEMPLATE_LOGGING_LEVEL = "restTemplateLoggingLevel";
```

Comment:
can you please rename this option to `loggingLevel` so that we don't need to add the same options for all other client libraries (e.g. jersey2, feign, etc) ?